projects/notebooks/visualizations/rhods-notebooks-performance/store/lts_parser.py

```diff
@@ -79,11 +80,16 @@ def generate_lts_metadata(results, import_settings):
 
 def generate_lts_results(results):
     results_lts = types.SimpleNamespace()
-
     results_lts.benchmark_measures = models.lts.BenchmarkMeasures.parse_obj(results.notebook_benchmark)
 
     return results_lts
 
+def generate_lts_regression(results):
+    results_lts = types.SimpleNamespace()
+    regression_results = [] if not results.regression_results else results.regression_results
+    results_lts = [matbench_models.RegressionResult.parse_obj(r) for r in regression_results]
+
+    return results_lts
```

Comment:
above, you have this:
```
lts_payload.regression = generate_lts_regression(results)
```

can you call the variable `regression`:

```suggestion
def generate_lts_regression(results):
    return [matbench_models.RegressionResult.parse_obj(r) for r in results.regression_results or []]
```

actually, you don't need the variable XD

what I wanted to say is:
```
...
lts_payload.regression = generate_lts_regression(results)
...

def generate_lts_regression(results):
    regression = types.SimpleNamespace()
    ...
    return regression
```
==> same variable / object attribute --> same name. Makes it easier to grep for where this object is used / defined